service-print/src/main/java/org/oskari/print/util/StyleUtil.java

```diff
@@ -69,11 +66,13 @@ public static PDPrintStyle getLineStyle (JSONObject oskariStyle) {
     }
     public static PDPrintStyle getPolygonStyle (JSONObject oskariStyle, PDResources resources) throws IOException {
         PDPrintStyle style = new PDPrintStyle();
-        JSONObject area = JSONHelper.getJSONObject(JSONHelper.getJSONObject(oskariStyle, "stroke"), "area");
-        setStrokeStyle(style, area);
+        JSONObject stroke = JSONHelper.optJSONObject(oskariStyle, "stroke", () -> new JSONObject());
+        JSONObject strokeArea = JSONHelper.optJSONObject(stroke, "area", () -> new JSONObject());
+        setStrokeStyle(style, strokeArea);
 
-        JSONObject fill = JSONHelper.getJSONObject(oskariStyle, "fill");
-        int pattern = JSONHelper.getJSONObject(fill, "area").optInt("pattern");
+        JSONObject fill = JSONHelper.optJSONObject(oskariStyle, "fill", () -> new JSONObject());
```

Comment:
You could skip all the lines up till setLabelStyle() if fill is null here? That way it would be more clear that setFillPattern() is not called. Though didn't check if style.setFillColor() does something with null param. optInt() returns 0 for null but since color will be null the setFillPattern() is not called. If the lines are just skipped anyone reading this code in the future wouldn't need to check what  optInt() and parseColors() do for nulls to figure out what is actually set for the style here.